flask_apscheduler/utils.py

```diff
@@ -146,4 +146,4 @@ def extract_timedelta(delta):
     w, d = divmod(delta.days, 7)
     mm, ss = divmod(delta.seconds, 60)
     hh, mm = divmod(mm, 60)
-    return w, d, hh, mm, ss
+    return w, d, hh, mm, ss
```

Comment:
Can you add a newline at the end, please?

Reply:
OK